sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -82,14 +82,14 @@
 
 
 # TODO (andreyvelich): We should add image tag for Storage Initializer and Trainer.
-STORAGE_INITIALIZER_IMAGE = "docker.io/kubeflow/storage-initializer"
+STORAGE_INITIALIZER_IMAGE_DEFAULT = "docker.io/kubeflow/storage-initializer"
```

Comment:
@varshaprasad96 Please can you submit PR to add the following change in the `constants.py`:
```
STORAGE_INITIALIZER_IMAGE = os.getenv(STORAGE_INITIAILIZER_IMAGE, "docker.io/kubeflow/storage-initializer")
TRAINER_TRANSFORMER_IMAGE = os.getenv(TRAINER_TRANSFORMER_IMAGE, "docker.io/kubeflow/trainer-huggingface")
```

That will allow users to quickly see the env they can modify, instead of searching in the `training_client.py`.

Reply:
I see!.. here we go: https://github.com/kubeflow/training-operator/pull/2268